SingularityUI/app/templates/taskDetail/taskMetadataTable.hbs

```diff
@@ -0,0 +1,76 @@
+{{#if data.taskMetadata}}
+    <div class="page-header">
+        <h2>
+            Metadata
+            <small>
+                <a data-action="expandToggle">
+                    {{#if expanded}}
+                        collapse
+                    {{else}}
+                        view
+                    {{/if}}
+                </a>
+            </small>
+        </h2>
+    </div>
+    <div class='{{#unless expanded}}hide{{/unless}}'>
+        <div class="table-container">
+            <table class="table table-striped">
```

Comment:
It's still kind of ugly, but it's at the bottom of the task page and starts collapsed. You wouldn't see it unless you want to find it, and even then it's very out of the way. So I think it's ok for now, we can iterate on it later if we want to.
![screenshot 2016-03-11 14 23 09](https://cloud.githubusercontent.com/assets/3210505/13713023/dcd3d7e6-e794-11e5-8e6f-17c369b7c546.png)